src/prpy/tsr/tsr.py

```diff
@@ -117,19 +122,17 @@ def is_valid(self, xyzrpy, ignoreNAN=False):
                     ((x - EPSILON) <= Bw_xyz[i, 1])
                     for i, x in enumerate(xyzrpy[0:3])]
 
-        # Unwrap rpy to [-pi, pi]
-        rpy = numpy.add(xyzrpy[3:6], pi) % (2*pi) - pi
         Bw_rpy = self.Bw[3:6, :]
         rpycheck = []
         for i in range(0, 3):
-            if (Bw_rpy[i, 1] > pi + EPSILON):
+            if (Bw_rpy[i, 0] > Bw_rpy[i, 1] + EPSILON):
                 # An outer interval
-                rpycheck[i] = ((rpy[i] + EPSILON) >= Bw_rpy[i, 0]) or\
-                              ((rpy[i] + EPSILON) <= Bw_rpy[i, 1] - 2*pi)
+                rpycheck[i] = (((rpy[i] + EPSILON) >= Bw_rpy[i, 0]) or
+                               ((rpy[i] + EPSILON) <= Bw_rpy[i, 1]))
```

Comment:
I believe @mkoval is referring to this particular line https://github.com/personalrobotics/prpy/pull/163#discussion-diff-36271837L121, where you've deleted the wrapping of `rpy` for some reason. I agree with him. That line is necessary, no?